### PR TITLE
Create cluster - validate account roles have managed policies attached

### PR DESCRIPTION
**Pull request by oriAdler**
Error out in case of missing managed policies for the account roles.

Related: [SDA-7904](https://issues.redhat.com/browse/SDA-7904)

Control plane role is missing the attached policy:
image

### CODE REVIEW

**Comment by oriAdler** [comment]
@zgalor @gdbranco @hunterkepley PTAL

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -2629,3 +2641,38 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 func getRolePrefix(clusterName string) string {
 	return fmt.Sprintf("%s-%s", clusterName, helper.RandomLabel(4))
 }
+
+func validateManagedAccountRoles(r *rosa.Runtime, prefix string, awsClient aws.Client) error {
```
Comment (zgalor) [inline cmd/create/cluster/cmd.go:2645]: My only though is if these 2 funcs belong in the `create cluster cmd` or maybe can also be used for other commands, and should be placed in some common pkg?

Comment (oriAdler) [reply cmd/create/cluster/cmd.go]: Good point.
Reused `ValidateAccountRolesManagedPolicies` from the `roles` package.
Moved `ValidateUnmanagedAccountRoles` to the `roles` package as well, though didn't find yet a place to reuse the function.

**Pull request merged**